packages/storybook-cmf/package.json

```diff
@@ -36,8 +36,8 @@
     "redux-saga": "^0.15.6"
   },
   "dependencies": {
-    "@storybook/addon-actions": "6.4.0-rc.3",
-    "@storybook/addons": "6.4.0-rc.3",
+    "@storybook/addon-actions": "^6.4.4",
+    "@storybook/addons": "^6.4.4",
```

Comment:
doesn't make sens to have storybook deps in both deps & dev deps

Reply:
the storybook plugin here depends on @storybook/addons but not addon-actions 
so I have removed it from here.
I dont remember why I have added @storybook/react as peer on this one but it is OK to have the same in peer and dev deps.
But none of them was in both deps and dev deps.